### PR TITLE
BeeRT

### PR DESCRIPTION
**Pull request by UwUPlusUwU**
Adds Bee ERT.

## About The Pull Request

Adds two outfits, one hardsuit, and a new response team.

## Why It's Good For The Game

We're Beestation yet we don't have Bee-related content. This changes now.

## Changelog
:cl:
add: The Bee Response Team have arrived.
/:cl:

### CODE REVIEW

**Comment by UwUPlusUwU** [comment]
I hope I fucking die, Qwert please smite me irl.

**Comment by SomeAngryMiner** [comment]
mind if i ask why is there an alien mosin in here?

**Comment by UwUPlusUwU** [comment]
Oh yeah, I forgot to take that out of the pack.
REEEEEEEEEEEEE

**Comment by qwertyquerty** [comment]
You may not have a personal icon file. Please sort the icons out into where they should belong, and clean up your PR from the other junk

**Comment by MarkSuckerberg** [comment]
I don't understand why all of this is in a separate directory... (and this is my personal opinion, I know it was a completely separate PR) the sharpened mop should be under spear.dm, the clothing should be under their respective .dm s (hardsuit.dm, ert.dm, etc.) I just don't like this form of semi-spaghettification, and don't see any need for it. Take this with a grain of salt though

File: code/modules/opo/bee.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	name = "Buzz Leader"
+	r_hand = /obj/item/nullrod/scythe/talking/chainsword
+	suit = /obj/item/clothing/suit/space/hardsuit/bee
+	backpack_contents = list(/obj/item/storage/box/engineer=1,
+		/obj/item/clothing/mask/gas/sechailer=1,
+		/obj/item/gun/energy/e_gun=1)
+
+//create antagonists
+
+/datum/ert/janitor
```
Comment (MarkSuckerberg) [inline code/modules/opo/bee.dm:52]: /datum/ert/janitor ?

File: code/modules/opo/bee.dm [hunk 2]
```diff
@@ -0,0 +1,59 @@
+//all names in this are WIP please send better names other than Buzz Trooper
+
+/obj/item/clothing/head/helmet/space/hardsuit/bee
+	name = "/improper BRT Hardsuit Helmet"
+	desc = "You feel faintly buzzed, and it isn't the omega weed..."
+	icon = 'code/modules/opo/opo.dmi'
```
Comment (MarkSuckerberg) [inline code/modules/opo/bee.dm:6]: Make sure to update this when you move the icons.

File: code/modules/opo/bee.dm [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	item_state = "bteamhardsuit"
+	icon_state = "bteamhardsuit"
+	armor = list("melee" = 35, "bullet" = 15, "laser" = 30, "energy" = 10, "bomb" = 10, "bio" = 100, "rad" = 50, "fire" = 75, "acid" = 75)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/bee
+
+//outfit code
+
+/datum/antagonist/ert/commander/bee
```
Comment (MarkSuckerberg) [inline code/modules/opo/bee.dm:24]: This should be in code/datums/ert.dm at the very least.

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by UwUPlusUwU** [comment]
Right, thanks.

File: code/modules/opo/bee.dm (hunk 1 above)
Comment (despong) [reply code/modules/opo/bee.dm:52]: janny ert

**Pull request closed without merging**